### DISCUSSION TITLE
OKR + GTD + Note => Logseq · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#148] [Announcements]
# OKR + GTD + Note => Logseq · BMPI

本文分享我使用Logseq实现人生管理系统的OKR、GTD与Note的功能。

https://www.bmpi.dev/self/okr-gtd-note-logseq/

**Comment by lrrgmes** [comment]
很实用的分享，在模仿设置时间四象限的待办事项，复制4个模块代码测试一直没有读取出来结果，有做abc优先级，带#okr关键的待办事项，看到能稍微指导下么
